src/scripts/main.js

```diff
@@ -1,5 +1,25 @@
 'use strict';
 
-// const pushNotification = (top, right, title, description, type) => {
-//   // write code here
-// };
+const pushNotification = (top, right, title, description, type) => {
+  const body = document.querySelector('body');
+
+  body.innerHTML = `
+    <div>
+      <h2 class="title">${title}</h2>
+      <p>${description}</p>
+    </div>
+    <h1 class="logo">MA</h1>
+    <script type="text/javascript" src="scripts/main.js"></script>
+  `;
+
+  const notification = document.querySelector('div');
+
+  notification.classList.add('notification');
+  notification.classList.add(`${type}`);
```

Comment:
Думаю класи краще одразу задвати якщо використовуєш innerHTML 
```suggestion
    <div class="notification ${type}">
      <h2 class="title">${title}</h2>
      <p>${description}</p>
    </div>
    <h1 class="logo">MA</h1>
    <script type="text/javascript" src="scripts/main.js"></script>
  `;

  const notification = document.querySelector('div');
```
І в середині .add() також не треба у рядок перетворювати якщо хочеш це залишити _.add(\`${type}\`) => .add(type)_ + якщо не помиляюся в .add() можна передавати кілька класів через кому щоб не дублювати виклик методу кілька раз або викликати ланцюгом .add().add()... спробуй)

Reply:
fixed